packages/app/src/components/Common/Dropdown/PageItemControl.tsx

```diff
@@ -66,12 +66,26 @@ const PageItemControlDropdownMenu = React.memo((props: DropdownMenuProps): JSX.E
     additionalMenuItemRenderer: AdditionalMenuItems, isInstantRename, alignRight,
   } = props;
 
+  // Change body class on change content width
+  const updateBodyClass = useCallback((isContainerFluid: boolean) => {
```

Comment:
You should put this function outside the component since it is better in terms of memory usage.